persistence/azure-table/configuration_code_astp_[3,).partial.md

```diff
@@ -67,3 +67,9 @@ In case installers are enabled, but there's a need to opt out from creating the
 
 snippet: EnableInstallersConfigurationOptingOutFromTableCreation
 
+### Partitioning
```

Comment:
I will try to reword this tomorrow. I think we need to make this clearer what the use case is. the intent was:

- enable partitioning and tx for customers that used previous version of the persistence and have sagas with messages in flight
- simple way to achieve partitioning and tx for conversations when sagas are involved to have the capability to precompute the saga id upfront so that business tx can participate even before the saga persister is at play

Reply:
Made some changes, please take a look